src/nxdoc/nuxeo-server/additional-services/bulk-action-framework/bulk-actions-directory.md

```diff
@@ -59,10 +59,7 @@ curl -X POST -H 'Content-Type: application/json' -u Administrator:Administrator
             "params":{
                 "query":"SELECT * FROM Document",
                 "action":"setProperties",
-                "parameters":{
-                    "dc:nature":"article",
-                    "dc:subjects":["art/architecture"]
-                }
+                "parameters":"{\"dc:nature\":\"article\",\"dc:subjects\":[\"art\/architecture\"]}"
```

Comment:
the `\` are expected to be displayed or is it to escape the `"`?

Reply:
They are expected to be displayed, yes